getting-started/background/translate_test.go

```diff
@@ -0,0 +1,98 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package background
+
+import (
+	"context"
+	"encoding/json"
+	"os"
+	"testing"
+
+	"cloud.google.com/go/firestore"
+)
+
+func deleteAll(projectID string) error {
```

Comment:
nit: I think it looks nicer to have tests at the top and helpers elsewhere/at the bottom. When I open a test file, I expect to see tests.

Feel free to ignore this if you feel differently.